File: Selenium/Activities/selenium_activity3.java
package selenium_session1;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class selenium_activity3 
{
	public static void main(String[] arg)
	{
		//create a WebDriver instance
		WebDriver driver = new FirefoxDriver();
				
		//go to a web page
		driver.get("https://training-support.net/webelements/login-form/");
		
		//do stuff on the page
		System.out.println("Title of the page is : " + driver.getTitle());
		
		//enter credentials in the username and password fields
		
		WebElement element = driver.findElement(By.xpath("//*[@id=\"username\"]"));
		element.sendKeys("admin");
		
		driver.findElement(By.xpath("//*[@id=\"password\"]")).sendKeys("password");
		
		//click submit button
		//driver.findElement(By.linkText("Submit")).click(); not working
		driver.findElement(By.xpath("/html/body/div/main/div/div/div/div/div[2]/form/button")).click();
	    
	    // Print the page title of the About Us page
	    System.out.println("New page title is: " + driver.getTitle());
	    
	    // Print the confirmation message
        String message = driver.findElement(By.xpath("//h1[contains(text(), 'Success')]")).getText();
        System.out.println("confirmation message : " + message);
	    
		//close the browser
		driver.quit();
	
	}
}

File: Selenium/Activities/selenium_activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

#create WebDriver instance
with webdriver.Firefox() as driver:

    #go to web page
    driver.get("https://training-support.net/webelements/login-form/")

    #do the stuff on the page
    pageTitle = driver.title
    print("Title of the page is : " + pageTitle)

    # enter credentials in the username and password fields
    driver.find_element(By.XPATH, "//*[@id=\"username\"]").send_keys("admin")
	
    driver.find_element(By.XPATH, "//*[@id=\"password\"]").send_keys("password")
	
	#click submit button
    #driver.find_element(By.LINK_TEXT, "Submit").click(); --not working
    driver.find_element(By.XPATH, "/html/body/div/main/div/div/div/div/div[2]/form/button").click()
    
    # Print the title of the new page
    pageTitle = driver.title
    print("New page title is: ", pageTitle)

	#Print the confirmation message
    message = driver.find_element(By.TAG_NAME, "h1")
    print("confirmation message is: ", message.text)
    

    
    #close the browser
    driver.quit()